src/PendingCalls/TestCall.php

```diff
@@ -168,6 +172,63 @@ public function skip(Closure|bool|string $conditionOrMessage = true, string $mes
         return $this;
     }
 
+    /**
+     * Sets the covered classes or methods.
+     */
+    public function covers(string ...$classesOrFunctions): TestCall
+    {
+        foreach ($classesOrFunctions as $classOrFunction) {
+            $isClass  = class_exists($classOrFunction);
+            $isMethod = function_exists($classOrFunction);
+
+            if (!$isClass && !$isMethod) {
+                throw new InvalidArgumentException(sprintf('No class or method named "%s" has been found.', $classOrFunction));
+            }
+
+            if ($isClass) {
+                $this->coversClass($classOrFunction);
+            } else {
+                $this->coversFunction($classOrFunction);
+            }
+        }
+
+        return $this;
+    }
+
+    /**
+     * Sets the covered classes.
+     */
+    public function coversClass(string ...$classes): TestCall
+    {
+        foreach ($classes as $class) {
+            $this->testCaseMethod->covers[] = new CoversClass($class);
+        }
+
+        return $this;
+    }
+
+    /**
+     * Sets the covered functions.
+     */
+    public function coversFunction(string ...$functions): TestCall
+    {
+        foreach ($functions as $function) {
+            $this->testCaseMethod->covers[] = new CoversFunction($function);
+        }
+
+        return $this;
+    }
+
+    /**
+     * Sets that the current test covers nothing.
+     */
+    public function coversNothing(): TestCall
+    {
+        $this->testCaseMethod->covers[] = new CoversNothing();
```

Comment:
Should this be the following so that you can't use a `coversNothing()` with a `covers()`? Not sure though, this should be fine. 👍🏻
```suggestion
        $this->testCaseMethod->covers = [new CoversNothing()];
```

Reply:
Mhh  you're right, I mean, you could, but it would be useless haha.  
Reading a second time the issue on PHPUnit I noticed that `@coversNothing` could be in **both** classes and methods and I think it would be better, for us, to always consider it as "per-method" to avoid undesired behaviours where the whole tests file covers nothing.

Now the issue is that PHPUnit reads by priority comments and attributes:

> PHPUnit 10 will first look for metadata in attributes before it looks at comments

For this reason if we use `#[CoversNothing]` then the `@test` comment will be ignored and the test is not executed. IMHO we have three paths:

1. Use `@coversNothing` as comment instead of attribute;
2. Transform `@test` comment into `#[Test]` attribute to use `#[CoversNothing]`, although a bit out of scope;
3. Use, for now, `@coversNothing`, when this is merged I'll open another PR to transform `@test` and `@coversNothing` into their respective Attributes (now or later, I mean).

What do you think should we do?